tests/ui/test_ui_tools.py

```diff
@@ -2321,7 +2324,7 @@ def test_main_view_generates_EDITED_label(
     @pytest.mark.parametrize(
         "to_vary_in_last_message, update_required",
         [
-            ({"sender_full_name": "Unique name (won't be in next message)"}, True),
+            ({"sender_id": "Unique id (won't be in next message)"}, True),
```

Comment:
Why is this updated? Does this behavior need updating too, as a follow-up?

Reply:
The function is merely checking update of author status so might not need that change. At that time I thought it'd need it since we were shifting from comparing based on `sender_full_name` to `sender_id`. Changing it back now